pkg/client/tables.go

```diff
@@ -171,6 +171,14 @@ func (c *Client) NewTableActionDirect(
 	}
 }
 
+func (c *Client) NewTableActionInDirect(
```

Comment:
I don't think this is the right naming. The term "indirect" would include both entries that point to a member or to a group. We should have `func (c *Client) NewTableActionMember(memberID uint32)` and `func (c *Client) NewTableActionGroup(groupID uint32)`.